sites/page-details/index.json-ld

```diff
@@ -265,15 +274,84 @@
 					"Déplacez le composant de date de modification hors de la grille et insérez-le juste avant la balise de fermeture <code>&lt;section></code>. Supprimez le <code>&lt;div class=\"col-xs-12\"></code> dans lequel il se trouvait."
 				]
 			}
+		},
+		{
+			"@id": "_:implement_pd_4to5",
+			"iteration": "_:iteration_pd_5",
+			"name": {
+				"en": "Migration from version 3.1 to 3.2",
+				"fr": "Migration de la version 3.1 vers 3.2"
+			},
+			"introduction": {
+				"en": "This implementation is meant for developers or publishers trying to migrate their page from using the page details version 3.1 to 3.2 (removal of \"col-lg-9\" class).",
+				"fr": "Cette implémentation est destinée aux développeurs ou aux éditeurs qui tentent de migrer leur page de la version 3.1 vers la version 3.2 (suppression de la classe «&nbsp;col-lg-9&nbsp;»)."
+			},
+			"instructions": {
+				"en": [
+					"Remove the \"col-lg-9\" class from the <code>&lt;div class=\"col-sm-8 col-md-9 col-lg-9\"&gt;</code> element."
+				],
+				"fr": [
+					"Supprimez la classe «&nbsp;col-lg-9&nbsp;» de l’élément <code>&lt;div class=\"col-sm-8 col-md-9 col-lg-9\"&gt;</code>."
+				]
+			}
```

Comment:
Please remove this implementation and rather add your instructions to the other one and call it "Migrating from 3.0 to 3.2".

```suggestion
		}
```

Reply:
Done locally, but I'll go ahead and merge this suggestion in any case :).

The "combined" 3.0 to 3.2 implementation instructions will be present in my next force-push.